<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentDetailsForm</class>
 <widget class="QWidget" name="AppointmentDetailsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Appointment Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="calendarLabel">
       <property name="text">
        <string>Calendar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="calendarCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="startLabel">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="startLayout">
       <item>
        <widget class="QDateEdit" name="startDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="startTimeLabel">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="startTimeEdit">
         <property name="displayFormat">
          <string>H:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="startSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="endLabel">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="subjectLabel">
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="placeLabel">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPlainTextEdit" name="descriptionEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="placeEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="subjectEdit"/>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="endLayout">
       <item>
        <widget class="QDateEdit" name="endDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="endTimeLabel">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="endTimeEdit">
         <property name="displayFormat">
          <string>H:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="endSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="buttonsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteAppointmentButton">
       <property name="text">
        <string>Delete Appointment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardChangesButton">
       <property name="text">
        <string>Discard Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveChangesButton">
       <property name="text">
        <string>Save Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calendarCombo</tabstop>
  <tabstop>startDateEdit</tabstop>
  <tabstop>startTimeEdit</tabstop>
  <tabstop>endDateEdit</tabstop>
  <tabstop>endTimeEdit</tabstop>
  <tabstop>subjectEdit</tabstop>
  <tabstop>placeEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>saveChangesButton</tabstop>
  <tabstop>discardChangesButton</tabstop>
  <tabstop>deleteAppointmentButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
